doc/site/core/sequence.markdown

```diff
@@ -18,6 +18,18 @@ If it returns `false`, stops iterating and returns `false`. Otherwise, returns
     [1, 2, 3].all {|n| n > 2} // False.
     [1, 2, 3].all {|n| n < 4} // True.
 
+### **any**(predicate)
+
+Tests whether any element in the sequence passes the `predicate`.
+
+Iterates over the sequence, passing each element to the function `predicate`.
+If it returns `true`, stops iterating and returns `true`. Otherwise, returns
```

Comment:
Yeah, I probably should have written more than one sentence there. I was intending to point out that the documentation wasn't accurately describing the behaviour of the implementation.